style.css

```diff
@@ -104,17 +120,17 @@ span {
 }
 
 .Proj-container h2 {
-  font-size: 2.5rem;
-  color: #172b4d;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
   letter-spacing: 0.1rem;
-  line-height: 2.8rem;
 }
 
 .proj-card {
   display: flex;
   flex-direction: column;
-  background-color: rgb(207, 207, 207);
-  border: 2px solid rgb(202, 202, 202);
+  background-color: #ebf0ee;
+  border: 2px solid #ebf0ee;
   border-radius: 10px;
 }
```

Comment:
you have done a great job in the card section but this is how your project cards should look like, ability to replicate a project exactly is great to have skill for a developer, Kindly add a margin around your card and use border-radius around it based on design

Kindly replicate all other designs on this page to match the template you chose

[optional] for semantic reason remove the `h1` in your welcome logo only one `<h1>` is needed on a page,only the `<a>` tag is okay for it.

![image](https://user-images.githubusercontent.com/43564324/159123819-fdda1e75-c2df-4935-b364-06dd93e7ee0f.png)